newIDE/app/src/UI/Accordion.js

```diff
@@ -135,7 +135,7 @@ type AccordionProps = {|
  */
 export const Accordion = React.forwardRef<AccordionProps, MUIAccordion>(
   (props, ref) => {
-    const { costlyBody, ...otherProps } = props;
+    const { costlyBody, noMargin, ...otherProps } = props;
```

Comment:
react complaining that this props was passed to the MUIComponent which didn't support it